recipes/pyside6qtads/meta.yaml

```diff
@@ -0,0 +1,64 @@
+ {% set version = "4.0.1" %}
+package:
+  name: pyside6qtads
+  version: {{ version }}
+
+source:
+  url: https://github.com/mborgerson/pyside6_qtads/archive/v{{ version }}.tar.gz
+  sha256: bb6c7efb88df26463ddfd0c25083a002e301495ec382333dd07b2155a77b2f57
+  patches:
+    - 0001-CMakeLists.patch
+    - 0002-generate_pyi.patch
+    - 0003-setup.patch
+    - 0004-bindings.patch
+    - 0005-init.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  ignore_run_exports_from:
+    # Just used for shiboken/pyside not used for python module
+    - clangdev  # [not osx]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clangdev
+    - ninja
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('libxext') }}           # [linux]
+
+  host:
+    - python
+    - cmake-build-extension
+    - qt6-main
```

Comment:
qt6 is still experimental at conda-forge. it is possible to build this with pyside2 and qt5 as a first pass?

Reply:
jschueller suggested that qt6/pyside6 could be in main branch and qt5/pyside2 created later. What should I do?